# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/tests/editing/other/insertparagraph-in-non-splittable-element.html
# Commit: 554d63dad12e
# Full Hash: 554d63dad12e4d1f2560e99de1fafac9a97da026
# Author: Masayuki Nakano <masayuki@d-toybox.com>
# Date: 2023-09-27 09:02:41
# Regressor Bug: 1851951
# File Overlap Count: 1
# Description:
#   Bug 1851951 - Make `HTMLEditor` refer computed `display` instead of the HTML default style at considering block or inline element r=m_kato
#   
#   Blink [1] and WebKit [2] refers `display-outside` value when they consider whether an
#   element is a block or an inline.
#   
# ==============================================================================

diff -r 97dc7be3b93c -r 554d63dad12e testing/web-platform/tests/editing/other/insertparagraph-in-non-splittable-element.html
--- a/testing/web-platform/tests/editing/other/insertparagraph-in-non-splittable-element.html	Wed Sep 27 02:50:33 2023 +0000
+++ b/testing/web-platform/tests/editing/other/insertparagraph-in-non-splittable-element.html	Wed Sep 27 03:01:26 2023 +0000
@@ -26,10 +26,7 @@
     selector: "caption",
     initial: "<div><table><caption>abc</caption><tbody><tr><td>abc</td></tr></tbody></table></div>",
     // <caption> can have paragraphs so that it should be handled as in a block.
-    expected: [
-      "<div><table><caption><br><div>abc</div></caption><tbody><tr><td>abc</td></tr></tbody></table></div>",
-      "<div><table><caption><div><br></div><div>abc</div></caption><tbody><tr><td>abc</td></tr></tbody></table></div>",
-    ],
+    expected: "<div><table><caption><br>abc</caption><tbody><tr><td>abc</td></tr></tbody></table></div>",
   },
   {
     selector: "col",
